libraries/engage/product/components/Rating/index.jsx

```diff
@@ -11,7 +11,7 @@ import RatingStars from '@shopgate/pwa-ui-shared/RatingStars';
 import RatingCount from '@shopgate/engage/reviews/components/Reviews/components/RatingCount';
 import { container } from './style';
 import connect from './connector';
-
+import { useWidgetSettings } from '../../../core/hooks/useWidgetSettings';
```

Comment:
please use '@shopgate/engage/core' to import 'useWidgetSettings', because there's an index file which exports among other things all hooks.